src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -88,11 +88,18 @@ public static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command)
                 return null;
             }
 
+            var commandType = command.GetType();
+            if (tracer.Settings.AdoNetExcludedTypes.Contains(commandType.FullName))
```

Comment:
NIT: You should first check if AdoNetExcludedTypes.Count is greater than 0 before calling Contains (fast-path)

Reply:
Thanks for spotting 😄 Done in 0760c6da8a21bf983fd90043d6f5056b838e26e2